server/storage.py

```diff
@@ -1,47 +1,68 @@
 import copy
+import uuid
+from typing import Optional
+from pydantic import BaseModel
+
 
 _GAMES_STORAGE = {
     1: {
         "id": 1,
+        "player_id_1": "",
+        "player_id_2": "",
         "status": "new",
         "current_move": "x",
+        "player_score_1": 0,
+        "player_score_2": 0,
         "field": [
-            [None, None, None],
-            [None, None, None],
-            [None, None, None],
+            ["", "", ""],
+            ["", "", ""],
+            ["", "", ""],
         ],
     },
 }
 
 
+def register_player():
+    global player_id_1, player_id_2
+
+    player_id_1 = uuid.uuid4()
+    player_id_2 = uuid.uuid4()
```

Comment:
The idea was to have arbitrary number of players. If you have only 2, it doesn't make sense to add registration functionality, you could create them in advance.

Also, this function will be overwriting players each time it's called. If this was done intentionally, then this is also wrong:
1. What if we want to conduct more than one game in parallel?
2. What if we want to register only one new player and have a game with an existing one?